composer.json

```diff
@@ -8,13 +8,13 @@
     },
     "minimum-stability" : "stable",
     "require": {
-        "php": ">=7.1",
-        "api-platform/core": "~2.5",
-        "ccmbenchmark/ting_bundle": "~3.0"
+        "php": ">=8.1",
+        "api-platform/core": "~3.1",
+        "ccmbenchmark/ting_bundle": "dev-feature/symfony/6"
```

Comment:
Version 3.6.0 of ting_bundle supporting SF 6 has been released. Could you change requirements to `"ccmbenchmark/ting_bundle": "^3.6"`